frontend/packages/kubevirt-plugin/src/selectors/event/filters.ts

```diff
@@ -12,8 +12,15 @@ import { VMILikeEntityKind } from '../../types/vmLike';
 
 type EventFilterFunction = (src: EventInvolvedObject) => boolean;
 
-const vmEventFilter = (vm: VMILikeEntityKind): EventFilterFunction => ({ kind, namespace, name }) =>
-  kind === VirtualMachineModel.kind && name === getName(vm) && namespace === getNamespace(vm);
+const vmEventFilter = (vm: VMILikeEntityKind): EventFilterFunction => ({
+  kind,
+  namespace,
+  name,
+}) => {
+  return (
+    kind === VirtualMachineModel.kind && name === getName(vm) && namespace === getNamespace(vm)
```

Comment:
did something change here, what am i missing ?

Reply:
Nothing changed with the logic. Prettier updated the formatting when I ran it.

Reply:
:-)

Reply:
p.s. 
I run:
`./node_modules/eslint/bin/eslint.js --ext .js,.jsx,.ts,.tsx,.json --color ../frontend/packages/kubevirt-plugin/src/ --fix`

I think it will not touch this kind of code...

p.p.s
can you undo the changes Prettier made, I like the shorter version best :-)